Lib/test/test_tools/test_makefile.py

```diff
@@ -51,7 +51,7 @@ def test_makefile_test_folders(self):
             if not dirs and not files:
                 continue
```

Comment:
And this is just redundant.

Reply:
Why? Skipping empty dirs is a part of the plan. 
Without these lines: `mkdir Lib/test/test_empty`

Will produce: `AssertionError: 'test/test_empty' not found in [...]`

Reply:
Does not the following check cover this case?

```pycon
>>> files= []
>>> all(filename.startswith('.') for filename in files)
True
```